src/Injector.php

```diff
@@ -10,68 +10,68 @@
 {
-    /**
-     * @var Definition\DefinitionInterface
-     */
-    protected $definition = null;
+    /** @var DefinitionInterface */
+    protected $definition;
 
-    /**
-     * @var ContainerInterface
-     */
-    protected $container = null;
+    /** @var ContainerInterface */
+    protected $container;
 
-    /**
-     * @var Resolver\DependencyResolverInterface
-     */
+    /** @var DependencyResolverInterface */
     protected $resolver;
 
-    /**
-     * @var ConfigInterface
-     */
+    /** @var ConfigInterface */
     protected $config;
 
-    /**
-     * @var string[]
-     */
+    /** @var string[] */
     protected $instantiationStack = [];
 
     /**
      * Constructor
      *
-     * @param ConfigInterface|null $config A custom configuration to utilize. An empty configuration is used
-     *      when null is passed or the parameter is omitted.
-     * @param ContainerInterface|null $container The IoC container to retrieve dependency instances.
-     *      `Laminas\Di\DefaultContainer` is used when null is passed or the parameter is omitted.
-     * @param Definition\DefinitionInterface $definition A custom definition instance for creating requested instances.
-     *      The runtime definition is used when null is passed or the parameter is omitted.
-     * @param Resolver\DependencyResolverInterface|null $resolver A custom resolver instance to resolve dependencies.
-     *      The default resolver is used when null is passed or the parameter is omitted
+     * @param ConfigInterface|null             $config A custom configuration to utilize. An empty configuration
+     *                                         is used when null is passed or the parameter is omitted.
+     * @param ContainerInterface|null          $container The IoC container to retrieve dependency instances.
+     *                                         `Laminas\Di\DefaultContainer` is used when null is passed or the
+     *                                         parameter is omitted.
+     * @param DefinitionInterface|null         $definition A custom definition instance for creating
+     *                                         requested instances. The runtime definition is used when null is
+     *                                         passed or the parameter is omitted.
+     * @param DependencyResolverInterface|null $resolver A custom resolver instance to resolve dependencies.
+     *                                         The default resolver is used when null is passed
+     *                                         or the parameter is omitted.
      */
     public function __construct(
-        ConfigInterface $config = null,
-        ContainerInterface $container = null,
-        Definition\DefinitionInterface $definition = null,
-        Resolver\DependencyResolverInterface $resolver = null
+        ?ConfigInterface $config = null,
+        ?ContainerInterface $container = null,
+        ?DefinitionInterface $definition = null,
```

Comment:
Please revert his change as it introduces BC break.

Reply:
those ? are added by cs-fix